docs/hub/spaces-changelog.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Spaces Changelog
+---
+
+<h1>Spaces Changelog</h1>
+
+## [2022-02-21] - Python versions
```

Comment:
@cbensimon are there any other changes from last 3 months we should add here? This seems very outdated